Jenkinsfile_CNP

```diff
@@ -26,7 +26,16 @@ def secrets = [
     secret('payments-queue-send-connection-string', 'PAYMENTS_QUEUE_WRITE_CONN_STRING')
   ]
 ]
-
+def prSecrets = [
+  's2s-${env}': [
+    secret('microservicekey-bulk-scan-orchestrator', 'S2S_SECRET')
+  ],
+  'bulk-scan-${env}': [
+    secret('idam-users-bulkscan-username', 'IDAM_USER_NAME'),
+    secret('idam-users-bulkscan-password', 'IDAM_USER_PASSWORD'),
+    secret('idam-client-secret', 'IDAM_CLIENT_SECRET')
```

Comment:
how does the existing functional tests work and why we need specifically for the  surname functional tests

Reply:
functional test were sending to the AAT. that is why there were no logs

Reply:
these are idam configs ? and not connection string ?

Reply:
nothing change for idam it is same as before. just exclude connection secrets for PR